electrum/notification_connector.py

```diff
@@ -0,0 +1,173 @@
+import dataclasses
+import functools
+from dataclasses import dataclass
+from hashlib import sha256
+from json.decoder import JSONDecodeError
+from typing import List
+
+import requests
+
+from electrum.logging import get_logger
+from electrum.wallet import Abstract_Wallet
+
+API_CONNECTION_STRING = 'https://btcv-notifcations-email.rnd.land/api'
```

Comment:
Typo: `notifcations` instead of `notifications`. Will it be redeployed?

Reply:
It's already changed